boards/arm/reel_board/doc/reel_board.rst

```diff
@@ -0,0 +1,434 @@
+| J2    | closed solder jumper  | open to use adapter for external target      |
++-------+-----------------------+----------------------------------------------+
+| J3    | open solder jumper    | close to pass UART TX to external adapter    |
++-------+-----------------------+----------------------------------------------+
+| J4    | open solder jumper    | close to pass UART RX to external adapter    |
++-------+-----------------------+----------------------------------------------+
+| J5    | closed solder jumper  | open to connect external adapter to nRF52840 |
++-------+-----------------------+----------------------------------------------+
+| J6    | closed solder jumper  | open to connect external adapter to nRF52840 |
++-------+-----------------------+----------------------------------------------+
+| J22   | open solder jumper    | close to start adapter in bootloader mode    |
++-------+-----------------------+----------------------------------------------+
+
+Adapter LEDs
+------------
+
++-------+-----------------------+--------------------------------+
+| Name  | Type                  | Usage                          |
++=======+=======================+================================+
+| D11   | green                 | flashes when adapter is active |
++-------+-----------------------+--------------------------------+
+| D14   | red                   | reserved                       |
++-------+-----------------------+--------------------------------+
+| D15   | yellow                | reserved                       |
++-------+-----------------------+--------------------------------+
+
+Programming and Debugging
+*************************
+
+Applications for the ``reel_board`` board configuration can be
+built and flashed in the usual way (see :ref:`build_an_application`
+and :ref:`application_run` for more details).
+
+Flashing
+========
+
+Build and flash
+applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Here is an example for the :ref:`hello_world` application.
+
+First, run your favorite terminal program to listen for output.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the reel board
+can be found. For example, under Linux, :code:`/dev/ttyACM0`.
+
+Then build and flash the application in the usual way.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: reel_board
+   :goals: build flash
+
+Debugging
+=========
+
+You can debug an application in the usual way.  Here is an example for the
+:ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: reel_board
+   :maybe-skip-config:
+   :goals: debug
+
+
+Testing the LEDs and buttons
+****************************
+
+There are 2 samples that allow you to test that the buttons (switches) and
+LEDs on the board are working properly with Zephyr:
+
+.. code-block:: console
```

Comment:
Change this code block into a bullet list:

    * :ref:`blinky-sample`
    * :ref:`button-sample`